Ex#1/ecasa_inv/src/main/java/com/avangenio/ecasa_inv/domain/Container.java

```diff
@@ -0,0 +1,51 @@
+package com.avangenio.ecasa_inv.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
+public class Container {
+	@Id
+	@JsonIgnore

```

Comment:
Using the entity as DTO is probably good for a small proyect but not a good practice.